sda-commons-server-auth/src/main/java/org/sdase/commons/server/auth/filter/JwtAuthFilter.java

```diff
@@ -42,28 +40,24 @@ private JwtAuthFilter() {}
 
   @Override
   public void filter(ContainerRequestContext requestContext) {
-    Span span =
-        tracer
-            .buildSpan("validateToken")
-            .withTag(COMPONENT, "JwtAuthFilter")
-            .withTag(AUTHENTICATED, false)
-            .start();
-
-    try (Scope ignored = tracer.scopeManager().activate(span)) {
+    Span span = tracer.spanBuilder("validateToken").setAttribute(AUTHENTICATED, false).startSpan();
+    //            .withTag(COMPONENT, "JwtAuthFilter")
```

Comment:
what happened to tags? use `setAttribute`?

Reply:
Forgot this 😅 the legacy `COMPONENT` tag would match the tracer name, I wanted to check in the guideline for a  better attribute but I think we do not need to know specifically which class is emitting this span, the tracer name `sda-commons.auth-bundle` should be enough